src/main/java/io/odpf/firehose/filter/FilterException.java

```diff
@@ -1,6 +1,6 @@
 package io.odpf.firehose.filter;
 
-public class FilterException extends Exception {
+public class FilterException extends RuntimeException {
```

Comment:
why did you change this to runtime? It should be a checked Exception,

Reply:
fixed